src/app/icd/server/ICDManager.h

```diff
@@ -174,6 +174,13 @@ class ICDManager : public ICDListener, public TestEventTriggerHandler
      */
     bool SupportsFeature(Clusters::IcdManagement::Feature feature);
 
+    // See ICDConfigurationData::SetModeDurations
+    CHIP_ERROR SetModeDurations(Optional<System::Clock::Milliseconds32> activeModeDuration,
```

Comment:
Adding a comment that this API doesn't currently garantee that spec compliance is respected if called at run time.